docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -290,6 +265,15 @@ Release
 
    Wait for the build to complete successfully.
 
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
```

Comment:
Doing it this way means that no software has to be installed on the host, but yes, this is a follow-up (https://clusterhq.atlassian.net/browse/FLOC-2514).